helm_deploy/hmpps-welcome-people-into-prison-ui/values.yaml

```diff
@@ -30,6 +30,7 @@ generic-service:
     REDIS_TLS_ENABLED: "true"
     TOKEN_VERIFICATION_ENABLED: "true"
     APPLICATIONINSIGHTS_CONNECTION_STRING: "InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY);IngestionEndpoint=https://northeurope-0.in.applicationinsights.azure.com/;LiveEndpoint=https://northeurope.livediagnostics.monitor.azure.com/"
+    ENABLED_PRISONS: 'BFI,BNI,CDI,DMI,EXI,EYI,HEI,HLI,LCI,LEI,LII,LPI,LWI,MHI,NMI,NWI,PNI,PVI,WCI,WSI,WWI'
```

Comment:
Didn't see much benefit in maintaining three separate lists now we're rolling out to more prisons. So I've moved this to the generic values file and enabled prisons are consistent across the environments.